content/pages/agenda.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Agenda"
+layout: "agenda"
+url: "agenda"
+---
+
+
+
+<style>
+td {vertical-align:top;}
+.agenda {
+border-width:2px;
+border-style:solid;
+border-color:black;
+border-collapse: collapse;
+width:60%;
+}
+
+th, td {
+  padding: 10px;
+}
+
+.agenda td {
+border-width:1px;
+border-style:solid;
+border-color:black;
+}
+
+.agendaLink {color: blue; text-decoration: none;}
+.agendaLink:hover {text-decoration: underline;}
+.agendaLink:active {color: black;}
+.agendaLink:visited {color: purple;}
+
+.timecontainer {width:20%;}
+.trainingcontainer {width:20%;}
+.descriptioncontainer {width:60%px;}
+
+</style>
+
+<!-- </style> -->
+
+<h1>Communication</h1>
+  <h4>Join us on <a href="https://join.slack.com/t/cascadiarconf/shared_invite/zt-1lu53059t-GAxQtzrwQhmo7BXE7YfC8w" target="blank_">Slack</a></h4>
+  <br>
+
+## June 21, 2024
+<br>
+
```

Comment:
Oh also I almost forgot since it's not explicit anywhere. 8-9 Registration and breakfast

Reply:
Done